services/ui-src/src/utils/other/time.ts

```diff
@@ -57,3 +59,24 @@ export const formatDateUtcToEt = (date: number): string => {
   // month + 1 because Date object months are zero-indexed
   return `${month + 1}/${day}/${year}`;
 };
+
+export const checkDateCompleteness = (date: string) => {
+  const month = parseInt(date.split("/")?.[0]);
+  const day = parseInt(date.split("/")?.[1]);
+  const year = parseInt(date.split("/")?.[2]);
+  const dateIsComplete = month && day && year.toString().length === 4;
+  return dateIsComplete ? { year, month, day } : null;
+};
```

Comment:
New method to check if a date is complete, because it was needed in a couple different places.